src/server/services/controller.ts

```diff
@@ -153,10 +153,12 @@ const NOTIFICATIONS = async (requestID: requestID, req: Request) => {
 
   const notifications: Array<MyNotification> = [
     ...tipNotifications,
-    ...chapterNotifications,
+    ...chapterNotifications, // can have tips as well
```

Comment:
Waar en hoe dan? Dit zou uitgesplitst moeten zijn.

Reply:
tja, zou moeten. Denk dat de fout zit in de mock data. In principe staat het sorteren los van uit welke stroom notifications een notification komt. Maar wel netjes de mock data te laten kloppen.

Reply:
de laatste van deze json wordt er in de development omgeving tussengedaan: src/server/mock-data/json/belasting.json